Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -1236,6 +1238,13 @@ void MainWindow::ShowMemcardManager()
   manager.exec();
 }
 
+void MainWindow::ShowCheatsManager()
+{
+  static CheatsManager* manager = new CheatsManager(this);
```

Comment:
Why static as opposed to being an instance-based dialog like everything else thus far? i.e. if it needs to have a longer lifespan than the function, instantiate as a class variable.